gradle.properties

```diff
@@ -6,4 +6,3 @@
 #
 # To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
 # warning at the top of this file.
-kotlin.incremental.useClasspathSnapshot=false
```

Comment:
mmh.. is this intentional? or if you didn't delete it then maybe the gradle plugin did?

Reply:
Yeah I didn't delete it the Gradle plugin did - as that's the case assume it's alright to go? 😅